lib/printer.js

```diff
@@ -225,9 +225,15 @@ function genericPrintNoParens(path, options, print) {
         return path.call(print, "program");
 
     case "Program":
-        return path.call(function(bodyPath) {
+        // if (n.directives) {
```

Comment:
I need help with directives - I think we need something like `printStatementSequence`? Just need to loop over the directives array